CONTRIBUTING.md

```diff
@@ -31,10 +31,13 @@ Please be aware of the following things when filing bug reports.
 
 If the bug report is missing this information then it'll take us longer to fix the issue. We will probably ask for clarification, and barring that if no response was given then the issue will be closed.
 
+## Creating a Pull Request
 
-## Submitting a Pull Request
+Submitting a pull request is fairly simple, just make sure it focuses on a single aspect and doesn't manage to have scope creep and it's probably good to go.
 
-Submitting a pull request is fairly simple, just make sure it focuses on a single aspect and doesn't manage to have scope creep and it's probably good to go. It would be incredibly lovely if the style is consistent to that found in the project. This project follows PEP-8 guidelines (mostly) with a column limit of 100 characters.
+### Formatting
+
+It would be incredibly lovely if the style is consistent to that found in the project. This project follows PEP-8 guidelines (mostly) with a column limit of 100 characters.
```

Comment:
This feels a bit too informal for contributing guidelines, perhaps something more like:

> We would greatly appreciate the code submitted to be of a consistent style with other code in the project.

Reply:
https://github.com/DisnakeDev/disnake/pull/601/commits/53893b007b09347b459d13c295552bd7fc6d37e5